packages/app/src/runner/SpecRunnerDropdown.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <Popover
-    :key="`${popoverKey}`"
+    :key="`${props.disabled}`"
```

Comment:
I wonder if this can just be `:key="props.disabled"`, should be the same thing 🤔 

Kinda late to the party, sorry I missed this.